README.rst

```diff
@@ -248,6 +248,33 @@ example CSP, this site uses the setting specified by the ``default-src``
 directive, which means that scripts can be loaded only from the
 originating server.
 
+Example 6
+~~~~~~~~~
+
+A web site administrator wants to allow embedded scripts (which might
+be generated dynamicially).
+
+.. code:: python
+
+    csp = {
+        'default-src': '\'self\'',
+        'script-src': '\'self\'',
+    }
+    talisman = Talisman(app,
```

Comment:
nit: can you line break at the opening `(` to avoid hanging indents?

Reply:
You mean
```python
foo = bar(
  arg=1
)
```

instead?

Reply:
yea